src/v3/test/integration/identify-with-password.test.tsx

```diff
@@ -251,6 +251,26 @@ describe('identify-with-password', () => {
     });
   });
 
+  it('should not pre-populate username into identifier field when set in cookie after an error', async () => {
+    const mockUsername = 'testuser@okta1.com';
+    jest.spyOn(cookieUtils, 'getUsernameCookie').mockReturnValue(mockUsername);
+    const {
+      user,
+      findByTestId,
+      findByText,
+    } = await setup({
+      mockResponse,
+      widgetOptions: { features: { rememberMyUsernameOnOIE: true } },
+    });
+    const submitButton = await findByText('Sign in', { selector: 'button' });
+    const usernameEl = await findByTestId('identifier') as HTMLInputElement;
+
+    await user.clear(usernameEl);
+    await user.click(submitButton);
```

Comment:
This isn't quite testing this scenario; you will want to clear the username, then type in a different user name (with a bad password). But doing so will also require you to mock the response to return an error, once the error is returned, you should be able to check the field to ensure the new username you entered matches what is in the field 😅 

A few moving parts with this, let me know if you have questions. 🙏